### ISSUE TITLE
Dimensions on Silkscreen-layer not supported => crash

### ISSUE THREAD

**Issue by nopeppermint** [openscopeproject/InteractiveHtmlBom#364]
Thanks for your nice tool.

I found a bug today, in my PCB I added some dimensions and apparently your tool do not proceed them correctly.
![html_bom_crash](https://user-images.githubusercontent.com/3403218/226842105-569de591-c4cc-4c23-b4be-a01560b2ab62.PNG)

It only happens if you place "dimensions" on the F.Silkscreen layer, if they are on a user layer it does not crash.

![image](https://user-images.githubusercontent.com/3403218/226843078-c6927a03-aa1c-4ca9-a835-902f20d3b522.png)

When I removed them from the PCB, there was no longer any crash.

Attached an example kicad 7 project where the bug is reproducible:
[dimensions_crash.zip](https://github.com/openscopeproject/InteractiveHtmlBom/files/11037875/dimensions_crash.zip)

Interactive BOM v2.5

My Kicad version:

```
Application: KiCad PCB Editor x64 on x64

Version: 7.0.1-41-g121ea14fe4, release build

Libraries:
	wxWidgets 3.2.2
	FreeType 2.12.1
	HarfBuzz 6.0.0
	FontConfig 2.14.1
	libcurl/7.88.1-DEV Schannel zlib/1.2.13

Platform: Windows 10 (build 19045), 64-bit edition, 64 bit, Little endian, wxMSW

Build Info:
	Date: Mar 20 2023 04:13:40
	wxWidgets: 3.2.2 (wchar_t,wx containers)
	Boost: 1.81.0
	OCC: 7.6.3
	Curl: 7.88.1-DEV
	ngspice: 39
	Compiler: Visual C++ 1934 without C++ ABI

Build settings:
	KICAD_SPICE=ON

```

**Comment by qu1ck** [comment]
Fix is now in v2.6

**Comment by nopeppermint** [comment]
thanks @qu1ck
